opendevin/llm/llm.py

```diff
@@ -114,10 +114,12 @@ def wrapper(*args, **kwargs):
             debug_message = ''
             for message in messages:
                 debug_message += '\n\n----------\n\n' + message['content']
-            llm_prompt_logger.debug(debug_message)
+            if config.get(ConfigType.DEBUG):
```

Comment:
Instead of here, we can set it here: https://github.com/OpenDevin/OpenDevin/blob/2be7e55303c3af06234cb7d356c62e85b617e133/opendevin/core/logger.py#L204, same for prompt and general. The logger level is checked by the file handler, which prints the message, starts and is about to print messages.